site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/impl/SiteManageGroupSectionRoleHandler.java

```diff
@@ -1640,11 +1640,17 @@ public String processImportedGroups() {
 			
 			//add all of the imported members to the group
     		for(String userId: importedGroup.getUserIds()){
-    			this.addUserToGroup(userId, group);
+    			try {
+    					String internalUserId = userDirectoryService.getUserId(userId);
+    					this.addUserToGroup(internalUserId, group);
+    				} catch (UserNotDefinedException e) {
+    					M_log.error("The eid: " + userId + "is invalid.");
+    				}
     		}
 
     		try {
     			siteService.saveGroupMembership(site);
+    			siteService.save(site);
```

Comment:
Why is this siteService.save necessary? Doesn't the siteService.saveGroupMembership handle this?